remove the tar archive to hopefully save some space

k-dominik opened the pull request:
i noticed that the tar file is still present in the ilastik docker containers. Removing it should save 800mb...

 # Submitting a Container
 
 (If you're requesting for a new container, please check the procedure described [here](https://github.com/BioContainers/containers#241-how-to-request-a-container).  
 
 ## Check BioContainers' Dockerfile [specifications](https://github.com/BioContainers/specs)  
 ## Checklist
 
 1. Misc
 - [ ] My tool doesn't exist in [BioConda](#make-sure-your-tool-isnt-already-present-in-bioconda)   
 - [ ] The image can be built   
 
 2. [Metadata](#check-biocontainers-dockerfile-metadata)  
 - [ ] LABEL base_image  
 - [ ] LABEL version
 - [ ] LABEL software.version  
 - [ ] LABEL about.summary  
 - [ ] LABEL about.home  
 - [ ] LABEL about.license   
 - [ ] MAINTAINER  
 
 3. Extra (optionals)
  - [ ] I have written tests in test-cmds.txt  
  - [ ] LABEL extra.identifier    
  - [ ] LABEL about.documentation   
  - [ ] LABEL about.license_file
  - [ ] LABEL about.tags   
 
 
 ## Check [BioContainers'](https://github.com/BioContainers/specs) Dockerfile metadata  


cc @sunyi000

osallou reviewed (commented):
Hi,
For automation, each pull request must match a single tool/version

No need to update old version, people will focus on latest one

k-dominik committed:
remove the tar archive to hopefully save some space

i noticed that the tar file is still present in the docker container.
Removing it should save 800mb...
